pkg/acceptance/bettertestspoc/assert/resourceparametersassert/task_resource_parameters_ext.go

```diff
@@ -0,0 +1,69 @@
+package resourceparametersassert
+
+import (
+	"strings"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+)
+
+func (u *TaskResourceParametersAssert) HasAllDefaults() *TaskResourceParametersAssert {
+	return u.
+		HasSuspendTaskAfterNumFailures(10).
+		HasTaskAutoRetryAttempts(0).
+		HasUserTaskManagedInitialWarehouseSize("Medium").
+		HasUserTaskMinimumTriggerIntervalInSeconds(30).
+		HasUserTaskTimeoutMs(3600000).
+		HasAbortDetachedQuery(false).
+		HasAutocommit(true).
+		HasBinaryInputFormat(sdk.BinaryInputFormatHex).
+		HasBinaryOutputFormat(sdk.BinaryOutputFormatHex).
+		HasClientMemoryLimit(1536).
+		HasClientMetadataRequestUseConnectionCtx(false).
+		HasClientPrefetchThreads(4).
+		HasClientResultChunkSize(160).
+		HasClientResultColumnCaseInsensitive(false).
+		HasClientSessionKeepAlive(false).
+		HasClientSessionKeepAliveHeartbeatFrequency(3600).
+		HasClientTimestampTypeMapping(sdk.ClientTimestampTypeMappingLtz).
+		HasDateInputFormat("AUTO").
+		HasDateOutputFormat("YYYY-MM-DD").
+		HasEnableUnloadPhysicalTypeOptimization(true).
+		HasErrorOnNondeterministicMerge(true).
+		HasErrorOnNondeterministicUpdate(false).
+		HasGeographyOutputFormat(sdk.GeographyOutputFormatGeoJSON).
+		HasGeometryOutputFormat(sdk.GeometryOutputFormatGeoJSON).
+		HasJdbcTreatTimestampNtzAsUtc(false).
+		HasJdbcUseSessionTimezone(true).
+		HasJsonIndent(2).
+		HasLockTimeout(43200).
+		HasLogLevel(sdk.LogLevelOff).
+		HasMultiStatementCount(1).
+		HasNoorderSequenceAsDefault(true).
+		HasOdbcTreatDecimalAsInt(false).
+		HasQueryTag("").
+		HasQuotedIdentifiersIgnoreCase(false).
+		HasRowsPerResultset(0).
+		HasS3StageVpceDnsName("").
+		HasSearchPath("$current, $public").
+		HasStatementQueuedTimeoutInSeconds(0).
+		HasStatementTimeoutInSeconds(172800).
+		HasStrictJsonOutput(false).
+		HasTimestampDayIsAlways24h(false).
+		HasTimestampInputFormat("AUTO").
+		HasTimestampLtzOutputFormat("").
+		HasTimestampNtzOutputFormat("YYYY-MM-DD HH24:MI:SS.FF3").
+		HasTimestampOutputFormat("YYYY-MM-DD HH24:MI:SS.FF3 TZHTZM").
+		HasTimestampTypeMapping(sdk.TimestampTypeMappingNtz).
+		HasTimestampTzOutputFormat("").
+		HasTimezone("America/Los_Angeles").
+		HasTimeInputFormat("AUTO").
+		HasTimeOutputFormat("HH24:MI:SS").
+		HasTraceLevel(sdk.TraceLevelOff).
+		HasTransactionAbortOnError(false).
+		HasTransactionDefaultIsolationLevel(sdk.TransactionDefaultIsolationLevelReadCommitted).
+		HasTwoDigitCenturyStart(1970).
+		HasUnsupportedDdlAction(sdk.UnsupportedDDLAction(strings.ToLower(string(sdk.UnsupportedDDLActionIgnore)))).
```

Comment:
probably because the same reason as here: https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/560ab6b444ac461684b0d25c68bcb2d55201aba8/pkg/acceptance/bettertestspoc/assert/objectparametersassert/gen/object_parameters_def.go#L89 (Snowflake lowercase default for one value and uppercase for other)

Reply:
Don't remember, but @sfc-gh-asawicki is most likely right and that's some kind of workaround (same as with "Medium" case).